src/Agent.Worker/Worker.cs

```diff
@@ -108,6 +108,17 @@ private void InitializeSecrets(JobRequestMessage message)
 
                 Trace.Verbose($"JobMessage: {channelMessage.Body}");
 
+                // Put system access token into variable.
```

Comment:
I saw your PR, i have the same thought, will rebase on yours, or comment this out till your stuff in.